examples/dynamo/torch_export_gpt2.py

```diff
@@ -56,6 +60,8 @@
     truncate_double=True,
     device=DEVICE,
     disable_tf32=True,
+    use_strong_types=True,
```

Comment:
Thoughts on calling this `use_strong_typing`?

Reply:
sounds good